src/test/java/library/view/ModeTest.java

```diff
@@ -0,0 +1,37 @@
+package library.view;
+
+import library.repository.BookRepository;
+import library.repository.FileBookRepository;
+import library.repository.InMemoryBookRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ModeTest {
+
+    @Test
+    @DisplayName("모드 코드로 모드를 찾을 수 있어야 합니다.")
+    void testFindByCode() {
+        Optional<Mode> normalMode = Mode.findByCode("1");
+        assertEquals(Mode.NORMAL, normalMode.get());
+
+        Optional<Mode> testMode = Mode.findByCode("2");
+        assertEquals(Mode.TEST, testMode.get());
+
+        Optional<Mode> invalidMode = Mode.findByCode("3");
+        assertFalse(invalidMode.isPresent());
+    }
+
+    @Test
+    @DisplayName("모드에 따라 도서 저장소를 가져올 수 있어야 합니다.")
+    void testGetBookRepository() {
+        BookRepository normalRepository = Mode.NORMAL.getBookRepository();
+        assertTrue(normalRepository instanceof FileBookRepository);
+
+        BookRepository testRepository = Mode.TEST.getBookRepository();
+        assertTrue(testRepository instanceof InMemoryBookRepository);
+    }
```

Comment:
```suggestion
    private static Stream<Arguments> modeStrategy() {
        return Stream.of(
            Arguments.of(Mode.NORMAL, FileBookRepository.class),
            Arguments.of(Mode.TEST, InMemoryBookRepository.class)
        );
    }

    @MethodSource("modeStrategy")
    @ParameterizedTest(name = "모드 {0}에 따라 도서 저장소를 가져올 수 있어야 합니다.")
    void testGetBookRepository(Mode mode, Class<?> expected) {
        BookRepository actual = mode.getBookRepository();
        assertThat(actual).isInstanceOf(expected);
    }
```

테스트 코드에는 단 하나의 Context를 검증하되 둘 이상의 파라미터가 존재하는 경우, JUnit5 프레임워크의 어노테이션을 적극 활용하는 것이 코드 가독성에 좋습니다.

Reply:
[6173dc2](https://github.com/prgrms-be-devcourse/java-library-management/pull/21/commits/6173dc24174fb42a06a7936190afbb8dddd10aef)
[44df1c5](https://github.com/prgrms-be-devcourse/java-library-management/pull/21/commits/44df1c5131dbc740cd92126e9db2c14eb23a672d)
반영했습니다!